src/core/global.rs

```diff
@@ -314,6 +314,24 @@ impl ServerError {
             ServerError::LicenseNoRemoteConnections => "The remote computer is not licensed to accept remote connections.".to_string(),
         }
     }
+
+    /// Some ServerErrors are really errors, whereas others are moreso
+    /// non-error reasons the server disconnected. This function returns
+    /// true for the former, false for the latter.
+    pub fn is_error(&self) -> bool {
```

Comment:
How are you planning to use this? What will be different from a user perspective when this is true or false?

Reply:
It will determine whether we log the message as an error or as info